modules/ixBidAdapter.js

```diff
@@ -258,17 +258,13 @@ export function bidToVideoImp(bid) {
 
   // if placement not already defined, pick one based on `context`
   if (context && !imp.video.hasOwnProperty('placement')) {
-    if (context === INSTREAM) {
-      imp.video.placement = 1;
```

Comment:
Technically this is accurate, the legacy `placement` doesn't distinguish the different types of "instream" presented in `plcmt`. I think what this bid adapter is missing is `plcmt` in addition to legacy placement

Reply:
they handle plcmt on line 257 above

however, removing so they can handle in their own pr